website/src/tabs/account/VerifyMessage.js

```diff
@@ -54,44 +62,27 @@ export const VerifyMessage = () => {
     if (aleo !== null) {
         const messageString = () => messageInput !== null ? messageInput.toString() : "";
         const signatureString = () => signatureInput !== null ? signatureInput.toString() : "";
-        return <Card title="Verify Message from Address" style={{width: "100%", borderRadius: "20px"}}
+        return <Card title="Verify a Message" style={{width: "100%", borderRadius: "20px"}}
                      bordered={false}>
             <Form {...layout}>
                 <Form.Item label="Address" colon={false}>
                     <Input name="address" size="large" placeholder="Address" allowClear onChange={onAddressChange}
                            style={{borderRadius: '20px'}}/>
                 </Form.Item>
+                <Form.Item label="Message" colon={false}>
+                    <Input name="Message" size="large" placeholder="Message" value={messageString()}
+                           style={{borderRadius: '20px'}} onChange={onMessageChange} />
+                </Form.Item>
+                <Form.Item
+                    label="Signature"
+                    colon={false}
+                    hasFeedback
+                    validateStatus={validateStatusSignature()}>
+                <Input name="Signature" size="large" placeholder="Signature" value={signatureString()}
+                       style={{borderRadius: '20px'}} onChange={onSignatureChange}/>
+                </Form.Item>
+
```

Comment:
The verification is a little hard to see on the message verification, maybe we make that green checkmark a little bigger?